ajax/libs/embedly-jquery/package.json

```diff
@@ -8,10 +8,7 @@
     "jQuery",
     "embed"
   ],
-  "license": {
-    "type": "BSD",
-    "url": "https://github.com/embedly/embedly-jquery/tree/master/LICENSE/"
-  },
+  "license":"BSD-3-Clause",
```

Comment:
Let's add a space between `:` and `"`.
Like from A to B.
- A
```
"license":"BSD-3-Clause",
```
- B
```
"license": "BSD-3-Clause",
```
Thank you.